addons/l10n_gr_edi/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'author': 'Odoo',
+    'name': 'Greece - MyDATA',
+    'version': '1.0',
+    'category': 'Accounting/Localizations',
+    'summary': """Connect to MyDATA API implementation for Greece""",
+    'description': """
+        MyDATA is a platform created by Greece's tax authority,
+        The Independent Authority for Public Revenue (IAPR),
+        to digitize business tax and accounting information declaration.
+    """,
+    'countries': ['gr'],
+    'depends': ['l10n_gr'],
+    'data': [
+        'data/ir_cron.xml',
+        'data/template.xml',
+        'security/ir.model.access.csv',
+        'views/account_fiscal_position_views.xml',
+        'views/account_move_views.xml',
+        'views/account_tax_views.xml',
+        'views/product_template_views.xml',
+        'views/report_invoice.xml',
+        'views/res_company_views.xml',
+        'views/res_partner_views.xml',
+    ],
+    'installable': True,
+    'auto_install': True,
```

Comment:
You need upgrade scripts to fill all new stored computed columns in existing models. E.g: `account.move.line.l10n_gr_edi_cls_*`
It can be done after this PR and forward ports are merged. Upgrade PRs cannot be tied to PRs targeting stable branches.

cc: @smetl

Reply:
As an example from another l10n:

```py
    def _auto_init(self):
        if not column_exists(self.env.cr, "account_move_line", "esg_emission_factor_id"):
            # Create the column to avoid computation during installation
            create_column(self.env.cr, "account_move_line", "esg_emission_factor_id", "int4")
            create_column(self.env.cr, "account_move_line", "esg_emission_multiplicator", "float8")
        return super()._auto_init()
```